### PR TITLE
Rdcc 316

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RDCC-316

### Change description ###
Removed status field in professional user, added deleted timestamp field to professional user alongisde other DM changes highlighted in the JIRA, updated tests to reflect changes

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V5__drop_unique_constraints_add_org_manager.sql [hunk 1]
```diff
@@ -6,4 +6,4 @@ ALTER TABLE organisation ADD COLUMN company_url varchar(512);
 
 ALTER TABLE organisation ADD COLUMN sra_id varchar(255);
 
-INSERT INTO PRD_ENUM (ENUM_CODE, ENUM_NAME, ENUM_TYPE, ENUM_DESC) VALUES (4, 'ORGANISATION_ADMIN', 'PRD_ROLE', 'Identifies the first user of an organisation');
```
Comment (akiohmcts) [inline src/main/resources/db/migration/V5__drop_unique_constraints_add_org_manager.sql]: We should not be making changes to previous migration scripts (even adding removing white-space) as it will be harder to read the git history for when a change was introduced and might even adversely effect Flyway. Please revert this change.

File: src/main/resources/db/migration/V6__adding_unique_constraints_dropping_status_add_new_field.sql [hunk 2]
```diff
@@ -0,0 +1,9 @@
+ALTER TABLE organisation ADD CONSTRAINT SRA_ID_UQ1 UNIQUE (SRA_ID);
+
+ALTER TABLE professional_user DROP COLUMN status;
+
+ALTER TABLE professional_user ADD COLUMN deleted timestamp;
+
+ALTER TABLE professional_user DROP CONSTRAINT EMAIL_ADDRESS_UQ1;
+
+ALTER TABLE professional_user ADD CONSTRAINT EMAIL_ADDRESS_UQ2 UNIQUE (EMAIL_ADDRESS, DELETED);
```
Comment (akiohmcts) [inline src/main/resources/db/migration/V6__adding_unique_constraints_dropping_status_add_new_field.sql]: Have you tried re-using the constraint name EMAIL_ADDRESS_UQ1? This would be better if it didn't present any problems.

File: src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java [hunk 3]
```diff
@@ -135,6 +135,11 @@ private void addPbaAccountToOrganisation(
 
         if (paymentAccounts != null) {
             paymentAccounts.forEach(pbaAccount -> {
+
+                if (pbaAccount == null || !pbaAccount.matches("(PBA|pba).*") || !pbaAccount.matches("^[a-zA-Z0-9]+$")) {
```
Comment (KOTLAPRA23) [inline src/main/java/uk/gov/hmcts/reform/professionalapi/service/impl/OrganisationServiceImpl.java]: Could you please add pattern matcher in the request field itself.
